apps/mobile/src/queries/market-data/stx-market-data.query.ts

```diff
@@ -0,0 +1,21 @@
+import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
+
+import { getMarketDataService } from '@leather.io/services';
+import { oneMinInMs } from '@leather.io/utils';
+
+export function createStxMarketDataQueryOptions() {
```

Comment:
I'm guessing this was a factory accepting arguments during the implementation, that were later deleted?
Also curious why `as const` – is the object being consumed elsewhere?